### ISSUE TITLE
Mirrored mode does not behave properly with 3 displays attached.

### ISSUE THREAD

**Issue by justMaku** [JohnCoates/Aerial#1268]
#### General troubleshooting tips

Before logging an issue please check that:
- [X] You have the latest version installed (There may be a beta version that fixes your issue), see here for the latest releases and bug fixes: https://github.com/JohnCoates/Aerial/releases
- [X] Your issue isn't already mentioned in our [issues](https://github.com/JohnCoates/Aerial/issues). You may find a workaround there or a similar request already made.
- [X] Your problem isn't mentioned in the [troubleshooting page](https://github.com/JohnCoates/Aerial/blob/master/Documentation/Troubleshooting.md).

If none of this fixes your issue, tell us about the problem you are experiencing or the feature you'd like to request.

#### Required information
In order to help us sort your issue, we ask that you provide the following information:
- [X] Mac model: MacBook Pro M1 Max
- [X] macOS version: Ventura 13.0
- [X] Monitor setup: 3 exactly the same displays side by side. 

If appropriate, please enable `Debug mode` and `Log to disk` in `Advanced` tab and replicate your bug, then attach the `AerialLog.txt` file (You can access this file through the Advanced tab).

#### Description of issue / Feature request

See the screenshot - I'd expect the first/left display to also be mirrored. 

<img width="1234" alt="Screenshot 2022-10-07 at 15 04 57" src="https://user-images.githubusercontent.com/2745656/194669135-b792b37c-cb3b-4dfc-9c2c-322c13cf09ee.png">

**Comment by glouel** [comment]
Hey @justMaku 

If I had to bet, your screens are ordered in macOS as
1) middle
2) right
3) left

And Aerial mirrors them in that order. 

Mirrored is something someone suggested for dual display, but it really falls down on complex cases (and yours isn't that complex, imagine 1 more monitor on top of your right screen). I'll try to see if I can improve it a bit but hard to say as there will always be weird edge cases.

**Comment by justMaku** [comment]
>Mirrored is something someone suggested for dual display.

Yup, that's me, but unfortunately due to \~reasons\~ I can not fix that :/

**Comment by glouel** [comment]
I had a look as I don't really remember much, the way it works is fairly basic, I need to implement something in Display Detection to take care of it. I'll try to have a look next week and keep you posted.

**Comment by justMaku** [comment]
No worries, and (more importantly) no rush!
